react/PlaceDetails.tsx

```diff
@@ -10,37 +10,40 @@ interface Props {
 
 const PlaceDetails: React.FC<Props> = ({ display = 'extended' }) => {
   const { address } = useAddressContext()
-  const summary = rules[address.country!].display[display]
-
-  const parseLineFragment = (
-    fragment: LineFragment,
-    index: number,
-    line: LineFragment[]
-  ) => {
-    const hasPreviousFragment = index > 0 && address[line[index - 1].name]
-    const hasNextFragment =
-      index + 1 < line.length && address[line[index + 1].name]
-    const hasDifferentDelimiter = fragment.delimiterAfter !== '-'
-    const shouldShowDelimiter = hasNextFragment ?? hasDifferentDelimiter
-
-    return address[fragment.name] ? (
-      <span key={fragment.name}>
-        {fragment.delimiter && hasPreviousFragment && (
-          <span>{fragment.delimiter}</span>
-        )}
-        <span className={fragment.name}>{address[fragment.name]}</span>
-        {fragment.delimiterAfter && shouldShowDelimiter && (
-          <span>{fragment.delimiterAfter}</span>
-        )}
-      </span>
-    ) : null
-  }
-
-  const parseLine = (line: LineFragment[], index: number) => [
-    <div key={index}>{line.map(parseLineFragment)}</div>,
-  ]
-
-  return <div>{summary.map(parseLine)}</div>
+  const displaySpec = rules[address.country!].display[display]
+
+  return (
+    <div>
```

Comment:
Can this `div` be a `React.Fragment`?

Reply:
_technically_ I think we wouldn't want to remove this div, because it would depend on the layout of the parent. maybe we can add a `flex flex-column` class here